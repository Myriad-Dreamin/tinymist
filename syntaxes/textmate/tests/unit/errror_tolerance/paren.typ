
#((show: body => 2) * body)

---