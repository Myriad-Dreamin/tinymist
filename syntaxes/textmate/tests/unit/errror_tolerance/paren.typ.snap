>
>#((show: body => 2) * body)
#^ source.typst keyword.control.hash.typst
# ^ source.typst meta.brace.round.typst
#  ^ source.typst meta.brace.round.typst
#   ^^^^ source.typst meta.expr.show.typst keyword.control.other.typst
#       ^ source.typst meta.expr.show.typst punctuation.separator.colon.typst
#        ^ source.typst meta.expr.show.typst
#         ^^^^ source.typst meta.expr.show.typst variable.parameter.typst
#             ^ source.typst meta.expr.show.typst
#              ^^ source.typst meta.expr.show.typst storage.type.function.arrow.typst
#                ^ source.typst meta.expr.show.typst
#                 ^ source.typst meta.expr.show.typst constant.numeric.integer.typst
#                  ^ source.typst meta.brace.round.typst
#                   ^^^ source.typst
#                      ^^^^ source.typst variable.other.readwrite.typst
#                          ^ source.typst meta.brace.round.typst
>
>---
#^^^ source.typst punctuation.definition.em-dash.typst